README.md

```diff
@@ -68,6 +73,23 @@ The rate limit is calculated as follows:
 
 This is a linear rate limit that increases over time with a maximum limit. This rate limit is derived from these values which can be set by and admin OR updated by the `CONTROLLER` role. The `CONTROLLER` updates these values to increase/decrease the rate limit based on the functionality within the contract (e.g., decrease the rate limit after minting USDS by the minted amount by decrementing `lastAmount` and setting `lastUpdated` to `block.timestamp`).
 
+## Trust Assumptions and Attack Mitigation
+Below are all stated trust assumptions for using this contract in production:
+- The `DEFAULT_ADMIN_ROLE` is fully trusted, to be run by governance.
+- The `RELAYER` role is assumed to be able to be fully compromised by a malicious actor. **This should be a major consideration during auditing engagements.**
+  - The logic in the smart contracts must prevent the movement of value anywhere outside of the ALM system of contracts.
+  - Any action must be limited to "reasonable" slippage/losses/opportunity cost by rate limits. **A malicious relayer should not be able to cause losses above 50bps of the principal in a 24 hours time frame.**
```

Comment:
Remove `malicious relayer should not be able to cause losses above 50bps of the principal in a 24 hours time frame`

I wrote that originally, but not sure if it makes much sense in the abstract. I think having the more open “reasonable” losses makes sense so we potentially get false positives vs missing something.